backend/src/db/providers/skillProvider.ts

```diff
@@ -194,6 +194,22 @@ const isProperTechnologies = async (techs: IUpdateCourseBody['skills']): Promise
   return checksPassed;
 };
 
+const skillsToCourseTechs = async (technologies: { skill: ObjectId; points: number }[]) => {
+  const techs = await Promise.all(
+    technologies.map(({ skill }) => {
+      return SkillModel.findOne({ name: skill });
+    }),
+  );
+
+  const techsForCourse = techs.map((currentSkill, index) => {
+    return {
+      skill: currentSkill?._id as ObjectId,
+      points: technologies[index].points,
+    };
+  });
```

Comment:
Make it one-line arrow function pls, if it has only return statement and nothing more